src/flow-util.c

```diff
@@ -137,10 +137,12 @@ void FlowInit(Flow *f, const Packet *p)
     if (PKT_IS_IPV4(p)) {
         FLOW_SET_IPV4_SRC_ADDR_FROM_PACKET(p, &f->src);
         FLOW_SET_IPV4_DST_ADDR_FROM_PACKET(p, &f->dst);
+        FLOW_SET_IPV4_TTL_FROM_PACKET(p, f);
```

Comment:
using this macro here is a bit confusing and inefficient too, can just be:
f->min_ttl_toserver = f->max_ttl_toserver = IPV4_GET_IPTTL((p));
Don't think the macro adds anything.

Reply:
I do agree.